docs/manual/guides/local/devilbox.en.md

```diff
@@ -0,0 +1,143 @@
+you have to create a separate directory here. The directory name then corresponds to the later vhost name. The directory 
+name `contao4` then results in` contao4.loc` for example.
+
+You have created a directory (e. g. `contao4`). Change to this directory and create a new subfolder `web`. Copy the 
+Contao Manager `.phar` file into this folder and rename the file to` contao-manager.phar.php`.
+
+{{% notice note %}}
+The domain suffix `.loc` is the default. However, this can be changed in the `.env` file via the entry `TLD_SUFFIX`.
+{{% /notice %}}
+
+
+## Installation via the Contao Manager
+
+Start `phpMyAdmin` in the Devilbox dashboard via `Tools\phpMyAdmin` and create a new database. Change then
+in the navigation to the page `Virtual Hosts`. Here you should see a list of your existing web projects
+and you can call them right away. You can now initiate the Contao installation via the Contao Manager. For example: `contao4.loc/contao-manager.phar.php`.
+
+The further procedure is then identical to the normal installation.
+
+
+## Installation via the command line
+
+By default, the PHP memory limit for the Devilbox's PHP container is too low and therefore must be previously configured for Composer.
+Change to the directory `cfg`. Did you configure the devilbox with PHP 7.3 in the `.env` file, make the following changes 
+accordingly in the directory `cfg\php-ini-7-3`. Create a file `memory_limit.ini` with the following entry:
+
+```bash
+[PHP]
+memory_limit = -1
+```
+
+Afterwards you have to restart the Devilbox. The Devilbox main directory contains the files `shell.sh` and` shell.bat`.
+So you can plug into the running Devilbox PHP container. Here are already [numerous tools](https://devilbox.readthedocs.io/en/latest/readings/available-tools.html) preinstalled. Also `composer`. After calling you are in the directory `shared\http`. To install e. g. Contao 4.8 in a directory `contao48` you just have to enter:
+
+```bash
+composer create-project contao/managed-edition contao48 4.8
+```
+
+Create a new database:
+
+```bash
+mysql -u root -h mysql -p -e 'CREATE DATABASE db_contao48;'
+```
+
+Afterwards you can leave the container via `exit` and call the Contao-Installtool.
+
+
+## The Contao-Installtool
+
+The information for the Contao-Installtool are basically identical. You only have to pay attention to the following entries:
+
+| Entry               | Value                 |
+|:--------------------|:----------------------|
+| **Host**            | mysql                 |
+| **Benutzername**    | root                  |
+| **Passwort**        | Do not enter a value  |
+
+{{% notice note %}}
+The user `root` with empty password is the Devilbox default setting. This could be changed in the Devilbox [configuration](https://devilbox.readthedocs.io/en/latest/support/faq.html#can-i-change-the-mysql-root-password). In this case, you must enter your values in the Contao Install-Tool.
```

Comment:
```suggestion
The user `root` with empty password is the Devilbox default setting. This could be changed in the Devilbox [configuration](https://devilbox.readthedocs.io/en/latest/support/faq.html#can-i-change-the-mysql-root-password). In this case, you must enter your configured credentials in the Contao Install Tool.
```